### PR TITLE
sweden.html

### PR DESCRIPTION
**Pull request by conerton**
What everyone thinking on requirements for the cities? Leo I see you have 10. I can add more if needed!

### CODE REVIEW

**Review by conerton — COMMENTED** [review]
I am trying to delete the "sweden html with temporary logo for company" file. I am working on it.

File: index.html [hunk 1]
```diff
@@ -11,35 +11,26 @@
     <main id="container">
```
Comment (leorondeau) [inline index.html]: Is this what you're talking about?

File: index.html [hunk 2]
```diff
@@ -11,35 +11,26 @@
...
-                <ul>Gothenburg</ul>
-                <ul>Malmö</ul>
+            <div>
+                List of cities
             </div>
 
-            <div class="landmarks">
-                <ul>The Drottingholm Palace</ul>
-                <ul>Stockholm Old Town</ul>
-                <ul>Kalmar Castle</ul>
```
Comment (leorondeau) [inline index.html]: This section looks cool

**Review by NotThatPatrickStewart — APPROVED** [review]

**Comment by NotThatPatrickStewart** [comment]
Test run, code was all removed. No changes to `index.html`

**Review by leorondeau — APPROVED** [review]

**Review by NotThatPatrickStewart — CHANGES_REQUESTED** [review]
Needs to include pull request info and testing instructions.

**Pull request merged**